website/the_stack/features.html

```diff
@@ -139,7 +139,7 @@ <h2 class="title">
 
 <div class="columns is-vcentered">
   <div class="column">
-<span style="color: #108bde">Blue</span>Banquise design was made to ensure full modularity.
+<span style="color: #108bde">Blue</span>Banquise can be seen as an Ansible collection of roles, which ensure full modularity.
```

Comment:
I would rephrase: ...as a set of Ansible roles, which ... 

This would be misleading to talk of Ansible collection on the website, because [Ansible Collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) exist and BlueBanquise is not compatible (yet?).

Reply:
I agree. Corrected.